### PR TITLE
Allow users to edit name

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

**Review by ShifaSZ — CHANGES_REQUESTED** [review]

File: seqr/views/apis/users_api.py
Comment (hanars) [reply seqr/views/apis/users_api.py:98]: I am happy to hold off on merging this until that PR is merged and then resolving any issues, or merging this one and resolving the issues on that PR. Unit tests will fail if this doesn't get updated correctly so theres no risk of something slipping, and so I'm happy to go in which ever order you approve

Comment (hanars) [reply seqr/views/apis/users_api.py]: the other PR is merged so this is resolved

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**
